tasks/populate-release-notes-images/populate-release-notes-images.yaml

```diff
@@ -76,6 +78,22 @@ spec:
                 containerImage="${deliveryRepo}@${digest}"
                 # purl should be pkg:oci/bar@sha256%3Aabcde?arch=amd64&repository_url=registry.redhat.io/foo
                 purl="pkg:oci/${deliveryRepo##*/}@${digest/:/%3A}?arch=${arch}&repository_url=${deliveryRepo%/*}"
+
+                uniqueTag=""
+                NUM_TAGS=$(jq length <<< "$tags")
+                for ((j = 0; j < NUM_TAGS; j++)) ; do
+                    tag=$(jq -rc --argjson j "$j" '.[$j]' <<< "$tags")
```

Comment:
Nit: The `-c` option of `jq` is redundant here. It's for compact json format, but here it's just one string, so nothing to make compact.